tests/testthat/_snaps/data_home_sidebar.md

```diff
@@ -28,6 +28,37 @@
       </ul>
       </div>
 
+---
+
+    Code
+      cat(data_home_sidebar(pkg))
+    Output
+      <div class='license'>
+      <h2 data-toc-skip>License</h2>
+      <ul class='list-unstyled'>
+      <li>NA</li>
```

Comment:
Why's there an NA here?

Reply:
Because there's no licence field in the DESCRIPTION used for this test, I fixed that.
I also opened https://github.com/r-lib/pkgdown/issues/1523